web-comp.js

```diff
@@ -0,0 +1,204 @@
+        category: "",
+        price: 379.99
+    },
+    {
+        id: 5,
+        imageURL: "images/product/mk17-img.jpeg",
+        altAlert: " it's an mk17",
+        name: "mk17",
+        description: "Long-range rifle with high accuracy and durable construction, chambered in 308 (7.62x51) allow to take down even the most armored target. Thanks to the fire switch we can use the mk17 even in full auto, be careful the recoil kick ass",
+        category: "",
+        price: 449.99
+    },
+    {
+        id: 6,
+        imageURL: "images/product/akm-img.jpeg",
+        altAlert: " it's an akm",
+        name: "akm",
+        description: "the one and only, the most famous rifle, the akm. chambered in 7.62x39 has served in numerous war becoming well known among the people.",
+        category: "",
+        price: 139.99
+    },
+];
+
+function getCatalogProductTemplate(productInfo){
+    const templateProduct = document.createElement("template");
+
+    templateProduct.innerHTML = `
+        <style>
+
+        .catalog-product{
+            background-color: white;
+            border: 1px solid black;
+            display: flex;
+            flex-direction: column;
+            flex-grow: 1;
+            max-width: 210px;
+            padding: 24px;
+        }
+
+        .squared-image-container{
+            aspect-ratio: 1;
+            border: 1px solid black;
+            width: 100%;
+        }
+
+        .product-image{
+            aspect-ratio: 1;
+            display: block;
+            max-height: 100%;
+            max-width: 100%;
+        }
+
+        .product-title{
+            margin: 0 auto;
+        }
+
+        .description-container{
+            border: 1px solid black;
+            height: 80px;
+            margin: 18px 0;
+            overflow: auto;
+            padding: 8px;
+        }
+
+        .product-price{
+            margin: 0 auto;
+            margin-bottom: 24px;
+        }
+
+        .product-actions{
+            display: flex;
+            justify-content: space-between;
+        }
+
+        .product-quantity-container{
+            align-items: center;
+            display: flex;
+            flex-wrap: nowrap;
+        }
+
+        .product-quantity{
+            margin: 0 16px;
+        }
+
+        </style>
+        <div class="products-container">
+            <div class="catalog-product">
+
+                <div class="squared-image-container">
+                    <img class="product-image" src="${productInfo.imageURL}" alt="${productInfo.altAlert}">
+                </div>
+
+                <h5 class="product-title">
+                    ${productInfo.name}
+                </h5>
+
+                <div class="description-container">
+                    ${productInfo.description}
+                </div>
+
+                <h5 class="product-price">
+                    ${productInfo.price}€
+                </h5>
+                    
+                <div class="product-actions">
```

Comment:
I removed the add button, I thought that would have been a good idea but I changed my mind.

Please re add it as "disabled"

Reply:
fixed

Reply:
fixed